torch/_subclasses/fake_impls.py

```diff
@@ -156,6 +156,9 @@ def constructors(fake_mode, func, *args, **kwargs):
     _, new_kwargs = normalize_function(
         func, args=args, kwargs=kwargs, normalize_to_only_use_kwargs=True
     )
+    if "names" in kwargs:
+        raise UnsupportedOperatorException("torch.compile doesn't support named tensors")
```

Comment:
TODO: should we raise the same exception in both places?

Reply:
I'm not sure what the difference is, but it looks consistent with how other exceptions are being thrown in the respective files